executor/src/wamr.rs

```diff
@@ -100,6 +101,19 @@ extern "C" {
 #[derive(Default)]
 pub struct WAMicroRuntime;
 
+macro_rules! early_terminate {
+    ($cond:expr, $msg:expr, $func:block) => {
+        if !($cond) {
+            reset_thread_context()?;
+            #[allow(unused_unsafe)]
```

Comment:
consider removal. after this pr, `wasm_runtime_***` will looks like "safe" if it's wrapped in early_terminate